internal/queue/queue.go

```diff
@@ -66,13 +79,22 @@ func (q *RabbitQueue) processActors(serviceManager *service.ServiceManager) {
 		},
 	}
 
-	err := q.StartConsumer(actorCQ)
-	if err != nil {
-		q.logger.Fatal("failed starting actor consumer", err)
+	for {
+		select {
+		case <-ctx.Done():
+			q.logger.Warn("actor context canceled")
```

Comment:
Should this be a warning? Is it a result of CTRL+C? Or is it a result of a failure?  If former, then I think debug is more appropriate.